plugins/qcheck-stm/src/reserr.ml

```diff
@@ -0,0 +1,31 @@
+module Ident = Gospel.Identifier.Ident
+
+type error =
+  | Value_is_a_constant of (Ppxlib.location * Ident.t)
+  | Value_return_sut of (Ppxlib.location * Ident.t)
+  | Value_have_no_sut_argument of (Ppxlib.location * Ident.t)
+  | Value_have_multiple_sut_arguments of (Ppxlib.location * Ident.t)
+
+let error_to_string = function
```

Comment:
We’ll eventually use the location info, I expect?
I’d like us to merge this back into `Warnings` at some point (not in this PR).

Reply:
Yes, this version is just enough for testing purposes. But it will be merged in `Warnings` where there is all what is needed to use the location information.